src/tests/ftest/util/pool_security_test_base.py

```diff
@@ -469,22 +444,22 @@ def verify_pool_acl_prim_sec_groups(self, pool_acl_list, acl_file, uuid):
                                               PERMISSIONS)
         self.modify_acl_file_entry(acl_file, grp_entry, new_grp_entry)
 
-        # dmg pool overwrite-acl --pool <uuid> --acl-file <file>
-        result = self.dmg.pool_overwrite_acl(uuid, acl_file)
-        self.log.info("  (8-4)dmg= %s", self.dmg)
+        # dmg pool overwrite-acl <pool name> --acl-file <file>
+        result = self.pool.overwrite_acl()
```

Comment:
(pylint-assignment-from-no-return) Assigning result of a function call, where the function has no return